rnjane/Blog-App
# #162658063 - Django project setup and Admin Login

What does this PR do?
Django project setup
Admin Login Feature
Description of Task to be completed?
Setup the proejct, and add authentication for admins.

How should this be manually tested?
Clone the repo at https://github.com/rnjane/Blog-App.git
CD into Blog-App folder
Install requirements - pip install -r requirements.txt
Run the app - python manage.py runserver
Create a superuser in your terminal.
Access the login API endpoint, and login with credentials used above.

You should get a token.
What are the relevant pivotal tracker stories?
#162658063
